src/index.ts

```diff
@@ -1,10 +1,10 @@
-import * as dotenv from "dotenv";
+// Before running the app, loads all the environment variables from a .env files
+// @see https://github.com/motdotla/dotenv/tree/v6.1.0#how-do-i-use-dotenv-with-import
+import "./env";
```

Comment:
this is not a good practice as there is a [side effect triggered by importing the module](https://palantir.github.io/tslint/rules/no-import-side-effect/) - the previous logic was much better (explicit)